spec/recipe.fmf

```diff
@@ -0,0 +1,109 @@
+story: |
+    As a user, I want tmt to generate a recipe file that can be reused to reproduce
+    a run with the exact same configuration, without requiring additional input.
+
+description: |
+    A recipe is a YAML file that captures a complete, static snapshot of a tmt
+    run after all dynamic evaluation has been resolved. It includes preprocessed
+    information about plans, tests, results, and run configuration. All environment
+    variables are evaluated, and git references are replaced with fixed commit hashes.
+
+    Generated recipe can be edited, manually or by using a script, which allows users
+    to select only a subset of discovered tests to be reexecuted.
+
+    .. code-block:: yaml
+
+        # Git repo configuration
```

Comment:
Which repository? There are many repositories, e.g. `discover` phases can clone many. Let's be specific.

Reply:
Maybe don't even call it `git`, can we find a better, more specific name, e.g. `something-repository`?

Reply:
`run-repository`?